components/ws-proxy/pkg/sshproxy/server.go

```diff
@@ -108,6 +136,7 @@ func (s *Server) HandleConn(c net.Conn) {
 	key, err := s.GetWorkspaceSSHKey(ctx, wsInfo.IPAddress)
 	if err != nil {
 		cancel()
+		s.TrackSSHConnection(wsInfo, "connect", err)
```

Comment:
i see this function can exit above too, should we track such exits as well?

Reply:
I mean that HandleConn can exit before reaching any "connect" track either with error or no. It seems we don't track such neither in logs or with analytics.

Reply:
Can we track it with prometheus? Another PR?